dask/dataframe/tests/test_dataframe.py

```diff
@@ -1609,13 +1610,19 @@ def test_dataframe_quantile(method, expected, numeric_only):
 
         assert (result == expected[1]).all().all()
 
-        with assert_numeric_only_default_warning(numeric_only):
+        with warnings.catch_warnings(record=True):
+            warnings.filterwarnings("ignore", category=FutureWarning)
+            # pandas issues a warning with 1.5, but not 1.3
             expected = df.quantile(axis=1, **numeric_only_kwarg)
         with assert_numeric_only_default_warning(numeric_only):
             result = ddf.quantile(axis=1, method=method, **numeric_only_kwarg)
-        assert_eq(result, expected)
 
-        with pytest.raises(ValueError), check_numeric_only_deprecation():
+        with check_numeric_only_deprecation():
+            assert_eq(result, expected)
```

Comment:
It looks like we're now warning at compute time -- is there a way we can avoid that?